recipes/fruit/all/patches/0001-conan-cmake.patch

```diff
@@ -1,8 +1,18 @@
 diff --git a/CMakeLists.txt b/CMakeLists.txt
-index d77de0c..338abb8 100644
+index d77de0c..6e385cc 100644
 --- a/CMakeLists.txt
 +++ b/CMakeLists.txt
-@@ -8,19 +8,8 @@ endif()
+@@ -5,22 +5,18 @@ if (POLICY CMP0054)
+     cmake_policy(SET CMP0054 NEW)
+ endif()
+ 
++if(NOT DEFINED CMAKE_CXX_STANDARD)
```

Comment:
You can set it directly in the Conan Cmake wrapper

Reply:
This policy was not added in the patch.
is set in fruit 3.4.0 CMakeLists.txt: https://github.com/google/fruit/blob/ece0f7218ce590ed6d4185326a31cb44bc451200/CMakeLists.txt#L4-L6

Do I have to move it to Conan CMake wrapper?

Reply:
Not the policy, the cmake standard.

Reply:
Sure.

Reply:
@uilianries Move it: 3ce64d9875e2af8c1ab083aa3cbc8794a0d97949

Reply:
Fix patch (removed ignore whitespace option in my git config): 8964f5430ada2e0e61ce275b72277bdc445a518a